tools/build_release.py

```diff
@@ -133,40 +160,57 @@
 
             test_spec["targets"][target_name] = toolchains
 
-        single_test = SingleTestRunner(_muts=mut,
-                                       _parser=parser,
-                                       _opts=options,
-                                       _opts_report_build_file_name=options.report_build_file_name,
-                                       _test_spec=test_spec,
-                                       _opts_test_by_names=",".join(test_names),
-                                       _opts_verbose=options.verbose,
-                                       _opts_only_build_tests=True,
-                                       _opts_suppress_summary=True,
-                                       _opts_jobs=options.jobs,
-                                       _opts_include_non_automated=True,
-                                       _opts_build_report=build_report,
-                                       _opts_build_properties=build_properties)
+        single_test = SingleTestRunner(
+            _muts=mut,
+            _parser=parser,
+            _opts=options,
+            _opts_report_build_file_name=options.report_build_file_name,
+            _test_spec=test_spec,
+            _opts_test_by_names=",".join(test_names),
+            _opts_verbose=options.verbose,
+            _opts_only_build_tests=True,
+            _opts_suppress_summary=True,
+            _opts_jobs=options.jobs,
+            _opts_include_non_automated=True,
+            _opts_build_report=build_report,
+            _opts_build_properties=build_properties
+        )
         # Runs test suite in CLI mode
-        test_summary, shuffle_seed, test_summary_ext, test_suite_properties_ext, new_build_report, new_build_properties = single_test.execute()
+        (
+            test_summary,
+            shuffle_seed,
+            test_summary_ext,
+            test_suite_properties_ext,
+            new_build_report,
+            new_build_properties
+        ) = single_test.execute()
     else:
         for target_name, toolchain_list in OFFICIAL_MBED_LIBRARY_BUILD:
-            if platforms is not None and not target_name in platforms:
-                print("Excluding %s from release" % target_name)
+            if platforms is not None and target_name not in platforms:
+                print(("Excluding %s from release" % target_name))
```

Comment:
Similar thing as above.
```suggestion
                print("Excluding {} from release".format(target_name))
```